packages/quantic/project.json

```diff
@@ -25,7 +25,6 @@
           "npm run babel:headless && npm run build:staticresources",
           "npm run build:doc"
         ],
-        "parallel": true,
```

Comment:
On a second thought, I don't think this line is helping to solve the issue because when we have `"parallel": true` the two commands ` "npm run babel:headless && npm run build:staticresources"` and `"npm run build:doc"` will be executed in parallel,  however inside the first command which is composed of two commands separated by an `&&`:
` "npm run babel:headless && npm run build:staticresources"`
I think the two commands will be executed sequentially because of the `&&`which chains commands in sequence

Reply:
I tried it right before adding .tmp to gitignore and it worked. Would you be more comfortable with this ?

```
"options": {
        "commands": [
          "npm run babel:headless",
          "npm run build:staticresources",
          "npm run build:doc"
        ],
        "parallel": true,
        "cwd": "packages/quantic"
      } 
```

Reply:
These two commands need to run sequentially

`"npm run babel:headless && npm run build:staticresources"`

That's why I had added them to the same line. And I don't see an issue running the `build:doc` command in parallel with those two to save some time?

Was the `build:doc` command creating an issue? Otherwise I would keep the `parallel: true` but definitely add the `.temp` folder to the gitignore and that should resolve the issue of having "left over" files no?

Reply:
The thing I suspect that this issue does not occur all the time, other wise we wouldn't have been able to merge the PR that introduces the ./temp folder,  so that's why I'm not sure about the effect of setting parallel tor `true`.

but we need to keep the `&&` between `npm run babel:headless` and the `npm run build:doc`.

Reply:
The issue is this.
https://github.com/coveo/ui-kit/actions/runs/11799966257/job/32869756535?pr=4670

Adding the .tmp to .gitignore fixes the issue
Removing parallel": true fixes the issue

Either one or both of these solutions works. I tested them both separately. Imo, simply adding the gitignore is bandaging the problem. The real problem is that the command gets cut off therefore not deleting the .tmp folder.

Reply:
For now we can simply make the change to the gitignore. Ill change the PR !